index.html

```diff
@@ -1,88 +1,89 @@
-<!DOCTYPE html
```

Comment:
I believe a minimal `<!DOCTYPE html>` is required to be W3 compliant.

Reply:
Yes, you are right, this minimal Doctype is still needed. I'll fix this.